### PR TITLE
Bug/alternative boost to brexit guidance

### PR DESCRIPTION
**Pull request by WillTaylor22**

### CODE REVIEW

File: activitystream/helpers.py [hunk 1]
```diff
@@ -21,9 +23,18 @@ def parse_results(response, query, page):
...
     content = json.loads(response.content)
-    results = serializers.parse_search_results(content)
-    total_results = content['hits']['total']
-    total_pages = int(ceil(total_results/float(RESULTS_PER_PAGE)))
+
+    if 'error' in content:
+        results = []
+        total_results = 0
+        total_pages = 1
+        logger.error('There was an error in /search', extra={
```
[the remark this answers is not in the document]
Comment (richtier) [reply activitystream/helpers.py]: `logger.error` does actually go to sentry :cool:

[the remark this answers is not in the document]
Comment (WillTaylor22) [reply activitystream/helpers.py]: I was thinking of this myself... is there a way to call sentry without halting the program?

File: dump.rdb [hunk 2]
```diff
@@ -0,0 +1 @@
+REDIS0006�ܳC�Z��V
```
[the remark this answers is not in the document]
Comment (richtier) [reply dump.rdb]: this is still here :) 
I think `git rm --cache dump.rdb` is the arbitrary command to make this file go away :laughing:

File: activitystream/helpers.py [hunk 3]
```diff
@@ -21,9 +23,18 @@ def parse_results(response, query, page):
...
+            'error': content['error'],
+        })
+    else:
+        results = serializers.parse_search_results(content)
+        total_results = content['hits']['total']
+        total_pages = int(ceil(total_results/float(RESULTS_PER_PAGE)))
```
[the remark this answers is not in the document]
Comment (WillTaylor22) [reply activitystream/helpers.py]: Excuse me, removed

File: dump.rdb (hunk 2 above)
[the remark this answers is not in the document]
Comment (WillTaylor22) [reply dump.rdb]: fingers crossed

File: activitystream/helpers.py [hunk 4]
```diff
@@ -1,13 +1,25 @@
...
 RESULTS_PER_PAGE = 10
+logger = logging.getLogger(__name__)
+
+client = Client(
+    **settings.RAVEN_CONFIG,
+    raise_send_errors=True,
+    install_sys_hook=False,
+    install_logging_hook=False,
+    transport=HTTPTransport
+)
```
Comment (richtier) [inline activitystream/helpers.py]: `from raven.contrib.django.raven_compat.models import client` will do this for you :)

Comment (WillTaylor22) [reply activitystream/helpers.py]: Right you are, thanks. I should have spotted the latter myself.

**Pull request merged**